events/wc_frostwolf_events.txt

```diff
@@ -1086,34 +1126,293 @@ narrative_event = { #Frostwolf settled
+	}
+	desc = {
+		trigger = { NOT = { character = event_target:target_frostwolf_leader } }
+		text = EVTDESC_WCFW_62_horde # Word arrives that the Frostwolf clan led by [target_frostwolf_leader.GetFirstName] has been completely decimated...
+	}
+	
+	picture = GFX_evt_orcish_horde
+	border = GFX_event_narrative_frame_war
+	
+	is_triggered_only = yes
+	hide_from = yes
+	
+	option = {
+		trigger = { character = event_target:target_frostwolf_leader }
+		name = EVTOPT_WCFW_62 # May the spirits curse you forever, [From.GetFirstName]!
+	}
+	
 	option = {
-		name = I_SEE
+		trigger = { NOT = { character = event_target:target_frostwolf_leader } }
+		name = INTERESTING
+	}
+	
+	after = {
+		# Create landless title for new orc (so he can be landless)
+		create_title = {
+			tier = DUKE
+			landless = yes
+			holder = event_target:target_new_orc
+			culture = event_target:target_frostwolf_butcher
+		}
+		
+		if = {
+			limit = { character = event_target:target_frostwolf_leader }
+			# Become adventurer orc if you're Frostwolf leader, they have a landless title so no game over
+			set_player_character = event_target:target_new_orc
+		}
+		
+		# Now butcher the whole frostwolf clan, abdicate to new orc + kill leader, killer unknown since done 'in secret'
+		event_target:target_frostwolf_leader = {
+			any_courtier = {
+				death = { death_reason = death_murder_unknown killer = event_target:target_frostwolf_butcher }
+			}
+			
+			# Destroy the Frostwolf tribe title and give the rest to the new orc
+			destroy_landed_title = d_frostwolf
+			abdicate_to = event_target:target_new_orc
+			death = { death_reason = death_murder_unknown killer = event_target:target_frostwolf_butcher }
+		}
+		
+		# Destroy the landless title so Dead Ravine becomes primary + make loyal vassal of Gul'dan/successor
+		event_target:target_new_orc = {
+			primary_title = {
+				activate_title = { title = THIS status = no }
+				destroy_landed_title = THIS
+			}
+			set_defacto_liege = event_target:target_frostwolf_butcher
+			opinion = { modifier = opinion_loyal_servant who = event_target:target_frostwolf_butcher }
+		}
+		
+		# Fire the event that speaks of the new inhabitants after three days (news has to travel, you know!)
+		73 = {
+			province_event = {
+				id = WCFW.163
+				days = 3
+			}
+		}
 	}
 }
 
+# delayer event
+province_event = {
+	id = WCFW.163
+	
+	is_triggered_only = yes
+	hide_window = yes
+	
+	trigger = {
+		event_target:target_frostwolf_butcher = {
+			is_alive = yes
+		}
+		event_target:target_new_orc = {
+			is_alive = yes
+		}
+	}
+	
+	immediate = {
+		any_player = {
+			limit = {
+				realm = { title = e_horde }
+			}	
+			narrative_event = { id = WCFW.63 }
+		}
+	}
+}
 
+# NEW INHABITANTS
+# Delayed event
+narrative_event = {
+	id = WCFW.63
+	title = EVTTITLE_WCFW_63
+	desc =  EVTDESC_WCFW_63 # The decimation of the Frostwolf clan left their lands uninhabited...
```

Comment:
> Not really a problem, but maybe another description in the case if butcher himself receive this event would be look better? Something like what you did in #768

Same with new orc

<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46576860/69395173-22df5480-0cef-11ea-8c56-ee55d9b62f58.png)


</details>